boards/witte/linum/linum.dts

```diff
@@ -146,6 +156,24 @@
 	d3ppre = <2>;
 };
 
+/* See reference manual (RM0433):
+ *   0b10: : HSE clock selected
+ */
+#define MCO1_SEL_HSE 2
```

Comment:
Already defined in  `include/zephyr/dt-bindings/clock/stm32h7_clock.h`. Please remove